source/agora/consensus/EnrollmentManager.d

```diff
@@ -343,6 +343,22 @@ public class EnrollmentManager
         return this.validator_set.getValidators(validators);
     }
 
+    /***************************************************************************
+
+        Get the number of active validators. Client code can use this between
+        clearExpiredValidators() calls to determine if the number of validators
+        has changed, and act accordingly.
+
+        Returns:
+            the number of active validators
+
+    ***************************************************************************/
+
+    public size_t validatorCount () @safe
```

Comment:
Can this be `const` ?

Reply:
Unfortunately not, because ValidatorSet.count() can't be const because it calls `db.execute` which cannot know in advance if the operation is a read or a modify.

Reply:
That's what I feared :(